# This represents the bump sensor data from the Neato 

uint8 leftFront
uint8 leftSide
uint8 rightFront
uint8 rightSide 
